### ISSUE TITLE
cmd-path not respected

### ISSUE THREAD

**Issue by tcolgate** [kubernetes-client/python#754]
The python client does not respect cmd-path in the gcp provider.

We abuse this feature to retrieve credentials for our non-gke clusters (hosted in AWS). This works well with k8s.io/client-go, but does not seem to be supported by the clients in the kubernetes-client org, with the exception of the nodejs client.

**Comment by jfrabaute** [comment]
We are having the same issue with a standard GKE cluster.
The kube config has "cmd-path" and the client is failing to refresh the access token.
Exception raised is:
```
google.auth.exceptions.DefaultCredentialsError: Could not automatically determine credentials. Please set GOOGLE_APPLICATION_CREDENTIALS or explicitly create credentials and re-run the application. For more information, please see https://cloud.google.com/docs/authentication/getting-started
```

Is there a plan to make this python client with with "cmd-path"?

**Comment by jfrabaute** [comment]
We're having the same problem with al our gke cluster which are using cmd-path.
I've just send a pull request to try to fix this: https://github.com/kubernetes-client/python-base/pull/168

**Comment by jfrabaute** [comment]
/remove-lifecycle stale

The fix is merged on python-base, new python client version not released yet.
Also, this PR might be needed as well to have this change fully operational: https://github.com/kubernetes-client/python-base/pull/169

**Comment by roycaihw** [comment]
Both fixes have been merged. They will be included in the upcoming 12.0.0a1 release

**Comment by jfrabaute** [comment]
It looks like the fixes are present in 11.0.0 release. @roycaihw can you confirm? If so, maybe we can close this issue.

**Comment by roycaihw** [comment]
@jfrabaute Thanks for pointing that out. You're right. 

/close
